torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20322,6 +20326,16 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
         ),
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.nn.functional.threshold",
+        torch_opinfo_name="nn.functional.threshold",
+        skips=(
+            # NotImplementedError: Could not run 'aten::where.ScalarOther' with arguments from the 'AutogradMeta'...
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_meta',),
```

Comment:
@mruberry just out of curiosity, what's the difference between marking expected test failures here vs in the UnaryUfuncInfo above?  (One difference is that only here seemed to work)

Reply:
These skips apply to `_refs.nn.functional.threshol`, while the other skips would apply to `nn.functional.threshold`